pandas/tseries/tests/test_resample.py

```diff
@@ -976,6 +976,21 @@ def test_resample_upsample(self):
 
         self.assertEqual(result.index.name, 'index')
 
+    def test_resample_how_method(self):
+        # GH9915
+        s = pd.Series(
+            {
+                Timestamp('2015-03-31 21:48:52.672000'): 2,
+                Timestamp('2015-03-31 21:49:52.739000'): 1,
+                Timestamp('2015-03-31 21:50:52.806000'): 3
+            }
+        )
+        result = s.resample("10S")
```

Comment:
specify this a ``s.resample('10s').mean()``; yes this works but hows a deprecation warning.
further just make an expected series and use ``assert_series_equal``